src/ol/interaction/ModifyPolygonBrush.js

```diff
@@ -103,6 +110,25 @@ class ModifyPolygonBrush extends Modify {
     }
   }
 
+  updateSketchRadiusByPressure_(event) {
+    if (this.sketchPoint_ && event.pointerEvent.pressure != 0) {
+      this.sketchPointRadius_ = getNewSketchPointRadius(event, this.sketchPointRadius_);
+      this.sketchRadius_ = Math.max(
+                             this.sketchPointRadius_ * event.pointerEvent.pressure,
+                             MIN_BRUSH_SIZE
+                           ) * event.map.getView().getResolution();
+    }
+  }
+
+  getSketchGeometry(event) {
+    var sketchGeometry  = this.sketchPoint_.getGeometry();
+    if (event.originalEvent.pointerType === 'pen') {
+      sketchGeometry = this.sketchPoint_.clone().getGeometry();
```

Comment:
This probably has a negative impact on performance (we spoke about this). Instead, create an invisible second sketchPoint that has the radius which was determined by pen pressure and update only the radius.